test-resources-hibernate-reactive/test-resources-hibernate-reactive-core/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id 'io.micronaut.build.internal.testcontainers-module'
+    id 'java-test-fixtures'
+}
+
+description = """
+Provides core support for Hibernate Reactive test resources.
+"""
+
+micronautBuild {
```

Comment:
I wonder why you use internal Micronaut plugins instead of ordinary Micronaut Application plugins?

Reply:
This isn't a test project, this is the core support for Hibernate Reactive. It's a Micronaut module.

Reply:
Ah I see